### PR TITLE
Add Go 1.19 to GHA

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták gliptak@gmail.com

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ jobs:
         go:
           - 1.17
```
Comment (milosgajdos) [inline .github/workflows/build.yml:24]: I wonder if we should remove `1.17` from GHA -- it's the last pre-generics version and might be causing all kinds of unnecessary problems for folks who started using generics 🤔

Comment (gliptak) [reply .github/workflows/build.yml]: updated

**Review by milosgajdos — APPROVED** [review]
LGTM. @thaJeztah PTAL

**Review by wy65701436 — APPROVED** [review]
lgtm

**Pull request merged**